ui/scss/component/_header.scss

```diff
@@ -1,3 +1,6 @@
+@import '../init/vars';
+@import '../init/mixins';
+
 .header {
```

Comment:
[Non blocking]
On a 1280 pixel wide screen, the wunderbar feels a bit squished.  Would be nice if able to maintain the original "fill remaining space, but until a certain max width", but not a big deal.

<img src="https://user-images.githubusercontent.com/64950861/146858699-5730202e-aad1-4868-87ea-d1d92e0587df.png" width="300">   <img src="https://user-images.githubusercontent.com/64950861/146858784-36b96c05-c823-46e8-9c4e-d34be0bb3c43.png" width="300">